src/main/java/frc/robot/subsystems/swerve/SwerveSubsystem.java

```diff
@@ -374,4 +411,36 @@ public Command pointTowardsTranslation(DoubleSupplier x, DoubleSupplier y) {
                         getPose().getRotation().getRadians(),
                         getVelocity().omegaRadiansPerSecond)));
   }
+
+  public Command runModuleSteerCharacterizationCmd() {
```

Comment:
What is our standard on having command or not in method names?

Reply:
i personally prefer using Cmd and probably should go through a pr to make it consistent. if you feel differently it doesnt matter a whole lot. iirc we started doing it because we needed swerve to have both a runVelocity and a runVelocityCmd

Reply:
Cool